src/backend/sev/builder.rs

```diff
@@ -47,6 +47,7 @@ impl kvm::Hook for Sev {
         let chain_packet = match generation {
             sev::Generation::Naples => Message::CertificateChainNaples(chain),
             sev::Generation::Rome => Message::CertificateChainRome(chain),
+            sev::Generation::Milan => Message::CertificateChainRome(chain),
```

Comment:
Is this supposed to be `CertificateChainMilan`?

Reply:
It's the same format. Maybe it should be renamed.

Reply:
It is fine as long as it isn't a mistake.

Reply:
@connorkuehl rename or add `CertificateChainMilan` ?

Reply:
@haraldh @connorkuehl Can we alias `Milan` => `Rome`? That would make things explicit.

Reply:
Hey, sorry, part of this conversation happened in chat. For posterity, here's what I said:

>  to be honest, I was trying to remember why there are different types at all, and I think it boils down to the fact that I needed to force serde to serialize/deserialize a union type by wrapping it in an enum. The important part is that some of the Naples certificates are Small (2048) and the Rome ones are "big" (4096).
> 
> I am partial to adding CertificateChainMilan, even if it is identical to Rome, because as far as the protocol goes, it labels the generation of certificate chain that the backend is sending. We use cbor, but if we used json the overall message type (https://github.com/enarx/koine/blob/master/src/attestation/sev.rs#L32) should come across as something like { "certificate-chain-milan": "blahbinarydata" } which I think is useful information when someone is developing client tooling

Would aliasing result in a Message type like this?: `{ "certificate-chain-rome": "actually a milan binary blob" }`

Reply:
fixed